src/txkube/testing/strategies.py

```diff
@@ -71,7 +71,17 @@ def dns_labels():
     ))
 
 # XXX wrong
-object_name = object_names = image_names = dns_labels
+object_name = object_names = dns_labels
+
+
+def image_names():
+    """
+    Strategy to build Docker image names.
```

Comment:
Think "strategy to" might be the hypothesis equivalent of "Test that ...".

I'd just say "Build docker image names."